File: trunk/ShadowFrameworkLite20/src/shadow/renderer/SFRenderable.java
package shadow.renderer;



public class SFRenderable {

	private SFProgramModuleStructures material=new SFProgramModuleStructures();
	
	
	public SFRenderable() {
		super();
	}

	public SFProgramModuleStructures getMaterialComponent() {
		return material;
	}

	public void setMaterialComponent(SFProgramModuleStructures material) {
		this.material = material;
	}

	
}

File: SFDataApplicationTestC++/shadow/renderer/SFRenderable.cpp

#include "shadow/renderer/SFRenderable.h"

namespace sf{

	SFRenderable::SFRenderable() {
		material=0;
	}

  	SFProgramModuleStructures* SFRenderable::getMaterialComponent() {
  		return material;
	}

  	void SFRenderable::setMaterialComponent(SFProgramModuleStructures* material) {
		this->material = material;
	}

}
